examples/src/integrationTest/java/org/apache/ignite/example/AbstractExamplesTest.java

```diff
@@ -52,7 +59,16 @@ public void startNode() {
 
         IgnitionManager.init(TEST_NODE_NAME, List.of(TEST_NODE_NAME), "cluster");
 
-        assertThat(ignite, willCompleteSuccessfully());
+        assertThat(igniteFuture, willCompleteSuccessfully());
+
+        ignite = (IgniteImpl) igniteFuture.get(1, TimeUnit.MILLISECONDS);
```

Comment:
you can use `join()` here, because the future is guaranteed to be completed after the assertion above

Reply:
Fix it.